packages/components/package.json

```diff
@@ -93,6 +93,5 @@
     "storybook-css-modules-preset": "^1.1.1",
     "tailwindcss": "^2.0.3",
     "typescript": "^4.3.5"
-  },
-  "gitHead": "8680dac1fa3c685e2a35f80d9ceb157f5f5343bd"
```

Comment:
https://github.com/lerna/lerna/issues/1880 indicates that `gitHead` is something Lerna should remove after publishing, and isn't intended to be committed.